docs/generate.html

```diff
@@ -130,6 +130,12 @@
     { id:       'money-gambling',
       versions: ['1.0', '1.1']
     },
+    { id:       'mood-intensity',
+      versions: ['1.1']
```

Comment:
is this 1.2?

Reply:
You're right, this would want a version bump. Upped to 1.2 and added a separate XML schema for that.